public/js/whiteboard.js

```diff
@@ -230,4 +226,10 @@ socket.on("dispatch", (dispatchResponse) => {
 // Refreshes the page allowing us to update the UI without ever touching the tv
 socket.on("refresh", () => window.location.reload());
 
-setInterval(() => updateDate(), 2000);
+const response = fetch('/locale')
+  .then((response) => response.json())
+  .then((data) => {
+    locale = data;
+    updateDate();
+    setInterval(() => updateDate(), 2000);
+});
```

Comment:
This can be handled without the `let locale;` up above, particularly because `locale` should be immutable. Maybe something like:
```
const locale = await fetch("/locale");
const [{lat}, {long}] = locale.json();
setInterval(() => updateDate(lat, long), 2000);
```
This may not be entirely correct syntax-wise, but I think it's a sleeker way to handle it than using `.then()`. I realize it involves slight restructuring of `updateDate()`, but it could be handled in other ways, too. You may be able to leave `lat` and `long` as globals `const`s and be done. Let me know your thoughts.